app/assets/javascripts/govuk_publishing_components/components/accordion.js

```diff
@@ -1,5 +1,85 @@
+/* global nodeListForEach */
```

Comment:
It's a little confusing that this code was removed and then restored in the same PR. Could this commit be squashed into an earlier commit?